### DISCUSSION TITLE
Lesson13: I get error at the begging.

### DISCUSSION THREAD

**Discussion by frank0532** [smartcontractkit/full-blockchain-solidity-course-py#1689] [General]
App.tsx code is as below:

```
import React from 'react';
import logo from './logo.svg';
import './App.css';
import { DAppProvider, ChainId } from '@usedapp/core';

function App() {
  return (
    <DAppProvider config={{
      supportedChains: [ChainId.Kovan, ChainId.Rinkeby, 1337]
    }}>
      <div>Hi!</div>
    </DAppProvider>
  );
}

export default App;
```

I get this Error:

ERROR in ./node_modules/@usedapp/core/dist/esm/src/abi/multicall/constants.js 1:0-31

Module not found: Error: Can't resolve 'ethers' in 'D:\syn\web3\defi-stake-yield-brownie\front_end\node_modules\@usedapp\core\dist\esm\src\abi\multicall'

**Comment by cromewar** [comment]
Hello @frank0532 
Could you check out the version you are using on package.json? 
There are a lot on changes since the course release 1 year ago, so using the newer version of the packages might be conflicting.
I highly recommend using the same versions as on the video.
